usng4j-impl/src/test/java/org/codice/usng4j/impl/CoordinateSystemTranslatorTest.java

```diff
@@ -1232,6 +1245,23 @@ public void testUsngToLatLon() throws ParseException {
     }
   }
 
+  @Test
+  public void testParsingInvalidLatitutedZonesToUsng() throws ParseException {
```

Comment:
Looking at this code, it seems to me that the only statement that can throw a `ParseException` is contained in a `try`-`catch` block. I think that this `throws` declaration can be removed.

Reply:
Done.